Documents/Technical Specifications/Technical.md

```diff
@@ -0,0 +1,494 @@
+    - [Branch](#branch)
+    - [Commit](#commit)
+    - [Pull Request](#pull-request)
+    - [Issue](#issue)
+- [Asset](#asset)
+  - [Environment](#environment)
+  - [Character](#character)
+    - [Sube](#sube)
+    - [NPC](#npc)
+  - [Plugin](#plugin)
+- [Gameplay](#gameplay)
+  - [Objective](#objective)
+  - [Camera](#camera)
+  - [Mechanics](#mechanics)
+    - [Movement](#movement)
+    - [Obstacle](#obstacle)
+    - [Sticky](#sticky)
+    - [Catapult](#catapult)
+    - [Teleport](#teleport)
+    - [Death](#death)
+      - [Death by falling](#death-by-falling)
+      - [Death by water](#death-by-water)
+      - [Death by heat](#death-by-heat)
+    - [Discuss](#discuss)
+  - [UI](#ui)
+    - [Main Menu](#main-menu)
+    - [Pause Menu](#pause-menu)
+    - [HUD](#hud)
+    - [Dialogue](#dialogue)
+      - [Help Dialogue](#help-dialogue)
+      - [Lore Dialogue](#lore-dialogue)
+  - [Audio](#audio)
+    - [Music](#music)
+  - [Animation](#animation)
+    - [Sube](#sube-1)
+    - [Catapult](#catapult-1)
+    - [Teleporter](#teleporter)
+  - [Cutscene](#cutscene)
+    - [Intro](#intro)
+- [Level](#level)
+  - [Level Design](#level-design)
+    - [Level 1](#level-1)
+    - [Level 2](#level-2)
+    - [Level 3](#level-3)
+  - [Lighting](#lighting)
+- [Softwares](#softwares)
+  - [Unreal Engine](#unreal-engine-1)
+  - [Epic Games Marketplace](#epic-games-marketplace)
+  - [Visual Studio](#visual-studio)
+  - [Git](#git-1)
+  - [Blender](#blender)
+
+</details>
+
+# General
+
+## Introduction
+
+The goal of this document is to define the technical specifications of the project.
+It will be used to define the requirements and the conventions of the project.
+As well as the tools we will use.
+
+## Glossary
+
+- **MVP**: Minimum Viable Product
+- **Asset**: A file that can be imported in Unreal Engine
+- **Perforce**: A version control system (Similar to Git)
+- **GitLab**: A version control system (Similar to Github)
+- **GitHub**: A version control system
+- **Blueprint**: A file that can be created in Unreal Engine
+- **Material**: A file that can be created in Unreal Engine
+- **Texture**: A file that can be created in Unreal Engine
+- **Animation**: A file that can be created in Unreal Engine
+- **Plugin**: A file that can be created in Unreal Engine
+- **Sube**: The main character of the game
+- **NPC**: Non-Playable Character
+- **UI**: User Interface
+- **HUD**: Heads-Up Display
+- **Dialogue**: A text that appears on the screen
+- **Cutscene**: A sequence of events that are not controlled by the player
+- **Level**: A map of the game
+- **Blur**: A visual effect that makes the image blurry
+- **Lore**: A story that is not directly related to the gameplay
+
+# Requirements
+
+## Unreal Engine
+
+We must use Unreal Engine 4.27.
+
+## Github
+
+We must use Github to manage our project.
+We also need to expand storage space for our project ( due to the size of the assets ).
+
+- Perforce doesn't solve the problem (Files still to big for Github).
```

Comment:
```markdown
- Perforce doesn't solve the problem (Files still to big for Github).
```
What do you mean by `Perforce doesn't solve the problem`.

perforce definition:
used to express necessity or inevitability.